frontend/fingertips-frontend/components/forms/SearchForm/index.tsx

```diff
@@ -36,6 +36,11 @@ export const SearchForm = ({
             },
           ]}
           data-testid="search-form-error-summary"
+          onHandleErrorClick={(targetName: string) => {
```

Comment:
I think given we've got our own logic here it would nice to have a test that validates clicking on the error link actually results in the indicator field being focused.